src/VerseSuggesting.ts

```diff
@@ -0,0 +1,153 @@
+import {
```

Comment:
Thanks again, 

I think this PR need a rebase, 

`src/VerseSuggesting.ts` was refactored to be under a folder.